CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable, unreleased changes to this project will be documented in this file.
 # 3.16.0 [Unreleased]
 
 ### Breaking changes
+- Change the Attribute - Product relation to decrease code complexity and make it easier to understand the relations - #13407 by @aniav
```

Comment:
Is it breaking change? It shouldn't have any impact on the Saleor users, right? It is only backend refactoring